xml/System.IO/DirectoryInfo.xml

```diff
@@ -2988,23 +2988,30 @@
       </ReturnValue>
       <Docs>
         <summary>Gets the parent directory of a specified subdirectory.</summary>
-        <value>The parent directory, or <see langword="null" /> if the path is null or if the file path denotes a root (such as "\\", "C:", or * "\\\server\share").</value>
+        <value>The parent directory, or <see langword="null" /> if the path is null or if the file path denotes a root (such as `\`, `C:\`, or `\\server\share`).</value>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- For a list of common I/O tasks, see [Common I/O Tasks](~/docs/standard/io/common-i-o-tasks.md).  
-  
-   
-  
-## Examples  
- The following example demonstrates referring to the parent directory of a specified directory.  
-  
- [!code-cpp[directoryinfoparent#1](~/samples/snippets/cpp/VS_Snippets_CLR/directoryinfoparent/CPP/directoryinfoparent.cpp#1)]
- [!code-csharp[directoryinfoparent#1](~/samples/snippets/csharp/VS_Snippets_CLR/directoryinfoparent/CS/directoryinfoparent.cs#1)]
- [!code-vb[directoryinfoparent#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/directoryinfoparent/VB/directoryinfoparent.vb#1)]  
-  
- ]]></format>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+> [!IMPORTANT]
+> In .NET Framework, `Parent` will return a relative path.
+> In .NET Core, `Parent` will return a fully qualified path.
+> To ensure consistent behavior across versions and explicit intent, call <xref:System.IO.DirectoryInfo.Name> on the returned <xref:System.IO.DirectoryInfo> when calling `Parent`.
```

Comment:
Although this produces consistent behavior, it only gets the name of the directory, without any path information, either relative or absolute. That doesn't strike me as being universally applicable guidance. Instead, I'd remove this sentence from the note, and include it in the remarks, and reword it as I suggested.
```suggestion

To ensure consistent behavior across versions and to make your intent explicit, retrieve the value of one of the following properties on the <xref:System.IO.DirectoryInfo> instance returned by `Parent`.
- <xref:System.IO.DirectoryInfo.Name>, which returns the simple name of the directory (such as `bin`).
- <xref:System.IO.DirectoryInfo.FullName>, which returns the absolute path of the directory. 
```

Reply:
Your suggestion makes sense. Thanks. I'll apply it.